### ISSUE TITLE
[🐛 Bug]: 4.11 in VB.net

### ISSUE THREAD

**Issue by kenaida99** [SeleniumHQ/selenium#12579]
### What happened?

I updated to 4.11 (from 4.7) and was able to use it in the debugging environment, but as soon as I published the update, none of the users were able to use the code. I kept getting an error message about the missing chromedriver.exe and it pointed me to the "https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/" address.

### How can we reproduce the issue?

```shell
sub main()
'this is the path containing the chromedriver.exe (taken from the JSON file containing the latest stable release)
Dim sPath As String = Environment.GetEnvironmentVariable("PATH")
    If sPath.Contains("C:\1stBrowser") = False Then
       Environment.SetEnvironmentVariable("PATH", sPath & ";" & "C:\1stBrowser\")
    End If

dim prv_drv As ChromeDriver
prv_drv = New Chrome.ChromeDriver(drvSvc, cOptions)
end sub

    Public Property drvSvc As Chrome.ChromeDriverService
        Get
            If IsNothing(_drvSvc) Then
                Try
                    _drvSvc = Chrome.ChromeDriverService.CreateDefaultService()
                Catch ex As Exception
                End Try

            End If
            Return _drvSvc
        End Get
        Set
            _drvSvc = Value
        End Set
    End Property
```

### Relevant log output

```shell
https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location/
```

### Operating System

Windows 10 & 11

### Selenium version

4.11.0

### What are the browser(s) and version(s) where you see this issue?

Chrome 116.0.5845.96 

### What are the browser driver(s) and version(s) where you see this issue?

Chromedriver  116.0.5845.96 

### Are you using Selenium Grid?

None

**Comment by titusfortner** [comment]
Why do you need to specify the default service? What happens if you remove that code?

**Comment by kenaida99** [comment]
Hi Titus, thanks for replying!

Users sometimes close down chrome and it causes a long delay before generating an error. If I use the driver service, I can still check the Title property which returns nothing instantly if the user has closed the browser.

I tried without using the driver service, however, and the same problem was caused.

I reverted to 4.7.0 and am now using 116 CFT (using the JSON endpoints) just fine. I’ll hold off going to 4.11 for now.

Thanks!
